janus_core/mlip_calculators.py

```diff
@@ -0,0 +1,101 @@
+"""Configure MLIP calculators.
+
+Similar in spirit with matcalc and quacc approaches
+- https://github.com/materialsvirtuallab/matcalc
+- https://github.com/Quantum-Accelerators/quacc.git
+"""
+
+from __future__ import annotations
+
+from typing import Literal
+
+from ase.calculators.calculator import Calculator
```

Comment:
check if we can get away with latest released ase, and if yes, put a note in README for the users for serious matters they shall install ase from git.